voorhoede/heads-up
# Feat: Fetch & Display Structured Data (Microdata, JSON-LD, RDFa).

## What change(s) were made
- Fetch Structured Data (Microdata, JSON-LD, RDFa.
- Add a new tab that displays the fetched data.
- It works for dom/body data & json/head data.
- It works on the Browser Extension & the Web App.

## How to test
- Go to: https://edition.cnn.com/2021/01/12/asia/indonesia-sriwijaya-plane-crash-intl/index.html or any other CNN page.
- Go to: The Structured Data tab.
- You should now see 2 code sections, one for Microdata (data is fetched from the dom/body), and the other for JSON-LD (data is fetched from the json/head).
## Related Trello ticket(s)
https://trello.com/c/6JkuzGIi
